htdocs/public/cron/cron_run_jobs.php

```diff
@@ -55,8 +55,12 @@
 $key = GETPOST('securitykey','alpha');
 if (empty($key))
 {
-	echo 'Securitykey is required. Check setup of cron jobs module.';
-	exit;
+	$key = argv[1];
```

Comment:
This script is not dedicated to be ran from CLI as it is into htdocs/public
You may make a confusion between script with same name into dir scripts/cron (dedicated to be ran as CLI only)